frontend/app/app.css

```diff
@@ -63,20 +71,29 @@ body {
   padding: 20px 0;
 }
 
+#mainContainer .row{ /* Fix materialize overflowing rows*/
+  margin-right: 0;
+  margin-left: 0;
+}
+#mainContainer h4, h5{
+  color: rgba(117, 59, 189, 1);
+}
+
+
 .clickable:hover {
   cursor: pointer;
 }
 
 .meme-img-small {
-  max-width: 300px;
+  max-width: 250px;
   width: 100%;
-  margin: 1em;
+  padding: 0.7em;
```

Comment:
Not sure if it's a good idea to add padding to the `meme-img-small` class. This class is used on the images on the front page which are also clickable links. Padding on a clickable element makes also the padding clickable which means that the clickable area is larger than the image itself.